docusaurus/docs/welcome/instructions.md

```diff
@@ -11,6 +11,24 @@ Keep in mind, part of adapting this boilerplate into a deliverable project is al
 
 ## 1. Tweak some project files
 
+### 1.1 Environment Configuration (.env)
+
+The boilerplate uses [`react-native-config`](https://github.com/luggit/react-native-config) to setup environment variables and expose them to javascript code. You can then access those variables and use them at any time when coding. It goes without saying that support for both iOS and Android platforms is available.
+
+By default, `react-native-config` will look for a `.env` file. Just create a `.env` file in the root of the project and setup your variables. An example is already provided in `.env.sample`, you can go check that out to get a feel of how it works. You can then run you project, in either platform, and you should be able to access the variables you just set (see the `Basic Usage` section of the docs [here](https://github.com/luggit/react-native-config#basic-usage)). 
+
+#### Multiple environments
+
+Most of the times there will be a need to have multiple environment files. In order to have this, a bit of a setup is needed. It's a little different for each platform, but nothing too much to worry about.
+
+For `Android`, you need to go to `app/build.gradle` and change the `envConfigFiles`. Scroll down a bit in the file, and you shall find them. The boilerplate comes with three already (debug, staging and production) all pointing to `.env`, but you can change them to your liking. Try setting up a variable in `.env.sample`, and set it to `devdebug` in `app/build.gradle`, see if it works. It probably should, and you're ready to create as many environment files as you need.
+
+For `iOS`, go to Xcode and open your project.  Then, go to `Product` > `Scheme` > `Edit Scheme...`. In the Build section, go to `Pre-actions`. There, change `.env` to whatever file you just created, or are about to create. Maybe try it out with the already existing `.env.sample`, see how it goes. Repeat this process for the other two schemes if you wish (staging and production), and you're done.
```

Comment:
There is no need to touch existing configurations. On both Android and iOS, react-native-config should look for `.env` by default regardless whether we're building for development, production, etc.

If the developer wants to have custom `.env.*` files for any particular reason, the following files are loaded up by the boilerplate's build process:
- .env.dev
- .env.prod
- .env.staging

Correct me if I'm wrong, but as far as my knowledge goes react-native-config works this way. The only thing I'm not 100% sure about concerning iOS is that if the file name written to `/tmp/envfile` does not exist, what will react-native-config do? Will it look for `.env` instead still? Can you confirm, please?

Reply:
Following our discussion [above](https://github.com/moxystudio/react-native-with-moxy/pull/18#discussion_r403095139), I guess we still can point the developer to the use of `ENVFILE` environment variable to specify a custom `.env`

Reply:
For multiple environment files, should we explain in the docs that the user will need to use the `ENVFILE` environment variable in the script to run the app, instead of using the pre-action scripts (iOS) or setting the `extConfigFiles` in build.gradel (Android) ?

Reply:
I'd drop the "Multiple environments" section entirely to avoid confusion. I think it is enough to state the following:
- `.env` is loaded by default
- Briefly mention `ENVFILE` environment variable can be used to specify a custom dotenv file as instructed at https://github.com/luggit/react-native-config#different-environments
For everything else, forward the user to `react-native-config`'s docs.

What do you think?

Reply:
That seems perfect to me actually. Should we leave this in instructions? Or do you want to still move it to What's Included?

Reply:
Yes, I think it makes more sense to move the whole document to "What's included > Environment variables". Unlike Next with MOXY, environment variables are not mandatory here. Therefore, I think we shouldn't give attention to this as far as project setup instructions go.